### PR TITLE
#2663 - Update TOS modal

### PR DESCRIPTION
**Pull request by drakkein**

### CODE REVIEW

File: app/modules/auth/selectors.ts [hunk 1]
```diff
@@ -62,7 +62,7 @@ export const selectIsVerifiedInvestor = createSelector(
 // TOS Related Selectors
 
 export const selectIsAgreementAccepted = (state: IAppState): boolean =>
-  !!(state.auth.user && state.auth.user.latestAcceptedTosIpfs !== undefined);
+  !!(state.auth.user && !!state.auth.user.latestAcceptedTosIpfs);
```
Comment (desfero) [inline app/modules/auth/selectors.ts]: do we need `!!` here? There is `!!` at the begging so it should be enought

File: app/components/modals/accept-tos-modal/AcceptTosModal.tsx [hunk 2]
```diff
@@ -63,7 +77,11 @@ export const AcceptTosModalInner: React.ComponentType<IStateProps & IDispatchPro
         </Button>
       </div>
       <div>
-        <Button onClick={() => onLogout(userType)} layout={EButtonLayout.SIMPLE}>
+        <Button
+          onClick={() => onLogout(userType)}
+          layout={EButtonLayout.INLINE}
+          size={ButtonSize.SMALL}
```
Comment (desfero) [inline app/components/modals/accept-tos-modal/AcceptTosModal.tsx]: This seems strange to provide size for `INLINE` button as this should mimic the look and feel of normal anchor tag

File: app/components/modals/accept-tos-modal/AcceptTosModal.tsx [hunk 3]
```diff
@@ -34,21 +40,29 @@ export const AcceptTosModalInner: React.ComponentType<IStateProps & IDispatchPro
 }) => {
   return (
     <section className="text-center">
-      <h1>
+      <img src={tosImg} className={styles.image} />
+      <Heading level={2} decorator={false}>
         {agreementChanged ? (
-          <FormattedHTMLMessage tagName="span" id="settings.modal.accept-updated-tos.title" />
```
Comment (desfero) [inline app/components/modals/accept-tos-modal/AcceptTosModal.tsx]: 👍

File: app/components/modals/accept-tos-modal/AcceptTosModal.tsx [hunk 4]
```diff
@@ -34,21 +40,29 @@ export const AcceptTosModalInner: React.ComponentType<IStateProps & IDispatchPro
 }) => {
   return (
     <section className="text-center">
-      <h1>
+      <img src={tosImg} className={styles.image} />
+      <Heading level={2} decorator={false}>
```
Comment (desfero) [inline app/components/modals/accept-tos-modal/AcceptTosModal.tsx]: 👍

File: app/components/modals/accept-tos-modal/AcceptTosModal.module.scss [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  margin: {
+    top: px-to-rem(10px);
+    bottom: px-to-rem(16px);
+    left: auto;
+    right: auto;
+  }
+}
+
+.download {
+  @include font-label;
```
Comment (desfero) [inline app/components/modals/accept-tos-modal/AcceptTosModal.module.scss]: hm, we should not change the font size for buttons. Maybe we can reuse of the layout we already have? Or we need to talk with designers

File: app/components/modals/accept-tos-modal/AcceptTosModal.module.scss [hunk 6]
```diff
@@ -0,0 +1,22 @@
+.image {
+  width: 270px;
+  margin-bottom: px-to-rem(28px);
+}
+
+.text {
+  @include font-label;
```
Comment (desfero) [inline app/components/modals/accept-tos-modal/AcceptTosModal.module.scss]: we should not change the size here as we have a bug with overall paragraph sizes. If you look at zeplin they all are set to 12px so this should be a global change IMO

File: app/components/modals/accept-tos-modal/AcceptTosModal.tsx [hunk 7]
```diff
@@ -34,21 +40,29 @@ export const AcceptTosModalInner: React.ComponentType<IStateProps & IDispatchPro
 }) => {
   return (
     <section className="text-center">
```
Comment (desfero) [inline app/components/modals/accept-tos-modal/AcceptTosModal.tsx]: This is a good use case to reuse our `Message` component. So all modals are consistent

Comment (drakkein) [reply app/components/modals/accept-tos-modal/AcceptTosModal.tsx]: :+1:

File: app/components/shared/buttons/Button.module.scss [hunk 8]
```diff
@@ -119,6 +119,7 @@ $_active-color: lighten($_hover-color, 20%);
 
     &#{$button}-small {
       @include font-label;
+      font-weight: $font-weight-bold;
```
Comment (desfero) [inline app/components/shared/buttons/Button.module.scss]: are you sure that's it's safe to change font-size of our small button? It's already used in a couple of places. Is this approved by designers?

File: app/components/modals/accept-tos-modal/AcceptTosModal.module.scss (hunk 5 above)
Comment (desfero) [reply app/components/modals/accept-tos-modal/AcceptTosModal.module.scss]: up

File: app/styles/overrides.scss [hunk 9]
```diff
@@ -58,6 +58,11 @@ textarea {
   }
 }
 
+// override default paragraph size
+p {
```
Comment (desfero) [inline app/styles/overrides.scss]: this i think should be done throughout sass variables

File: app/components/shared/buttons/Button.module.scss (hunk 8 above)
Comment (drakkein) [reply app/components/shared/buttons/Button.module.scss]: It's verified and accepted by @Louisswryan

**Pull request merged**